mobileClient/components/Video.js

```diff
@@ -42,7 +42,7 @@ export default class Video extends React.Component {
   }
   onFinished = () => {
     this.setState({playVideo: false})
-    this.props.onVideoWatched({event: {state: 'ended'}})
+    this.props.onVideoWatched && this.props.onVideoWatched({event: {state: 'ended'}})
```

Comment:
do we need this check? when this.props.onVideoWatched would not be passed now, since you added the props in the parent component? Does this happen in the lectures list?

Reply:
As far as I understand `onVideoWatched` is an optional callback used by parent components to know when video playback stops. Because of this optional nature we need to ensure that `Video` component won't break in any case. This is good practice imo. 
`Video` is also used in `Lectures` where `onVideoWatched` porp is not passed.